### ISSUE TITLE
[Feature Request] Display upload or download progress

### ISSUE THREAD

**Issue by ericschdt** [dino/dino#754]
Downloading or uploading larger files may take a while especially depending on the bandwith. Therefore it would be convenient to see amount of data sent / received vs the total file size.

**Comment by catap** [comment]
@mesonium I just tried fd915ac76fdb5d3bf5a2d9d37476ed7cf0f16bd2 with ~8 mb video, and no progress bar, and it looks like not upload at all

**Comment by catap** [comment]
@licaon-kter indeed, uploading this video as archive works as expected, with progress bar.
